package.json

```diff
@@ -45,7 +45,7 @@
   },
   "scripts": {
     "lint": "eslint --plugin markdown --ext js,md .",
-    "test": "mocha --require test/support/env --reporter spec --check-leaks --bail test/",
+    "test": "mocha --require test/support/env --reporter spec --check-leaks --bail test/ --globals __core-js_shared__",
```

Comment:
Where does this global leak come from?

Reply:
No idea. I tried many versions - it seems to happen only on very old node versions, and even when we just have the brotli test in there, which is no different than other tests. I guess the *theoretic* call to `zlib.createBrotliDecompress` confuses it. There's no brotli on those versions.

Reply:
Does the leak happen on CI? If it does not happen on CI, just remove it. If it does, then that would give the answer to provide here, but in order to accept this it either needs to happen on CI or there needs to be a clear explanation as to why this is here.

Reply:
Yes, it happens on the CI only. And on old node versions I can't install, so can't see if they happen locally.

Reply:
Which CI versions? All of them or only a subset?

Reply:
A bunch of CI versions, I think below node 6, but don't remember precisely.

Reply:
Can you link me to the build? Since you keep squashing it loosing the history. If you cannot not, how about push up removing this so I can see precisely which ones? Because I can help look into why it is happening if I know where to start looking.

Reply:
Yeah, pushed, let's see.

Reply:
Cool. I believe it is running the commit without this, and no failures so far. Build: https://travis-ci.org/github/expressjs/body-parser/builds/706944258

Reply:
Yeah, seems fine :-) I guess it was a combination of something that caused older node versions to maybe pull in something else from core-js, if you're using babel or something.